lab-kyle/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "y",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab-05-tcp-chat-server ======",
+  "main": "server.js",
+  "scripts": {
+    "test": "mocha"
```

Comment:
Shouldn't be mocha since you have no tests, do you need a test script for this assignment?